product/src/main/java/com/yas/product/viewmodel/product/ProductExportingDetailVm.java

```diff
@@ -0,0 +1,25 @@
+package com.yas.product.viewmodel.product;
+
+import java.util.List;
+
+public record ProductExportingDetailVm(
+        long id,
+        String name,
+        String shortDescription,
+        String description,
+        String specification,
+        String sku,
+        String gtin,
+        String slug,
+        Boolean isAllowedToOrder,
+        Boolean isPublished,
+        Boolean isFeatured,
+        Boolean isVisible,
+        Boolean stockTrackingEnabled,
+        Double price,
+        Long brandId,
+        String brandName,
+        String metaTitle,
+        String metaKeyword,
+        String metaDescription) {
+}
```

Comment:
Fix End of line issue as github recommended
![image](https://user-images.githubusercontent.com/103926328/227096008-f5b2bc30-fbd0-4ea2-849b-b089fe9a26c2.png)